src/adagrams.js

```diff
@@ -1,15 +1,107 @@
+'use strict';
+
+import { letterPool, scoreChart } from './constants.js';
+
+
+const createLetterPoolList = (letterPoolDict) => {
+  //Function has one parameter, a dictionary, and returns a list of letters.
+  let letterPoolList = [];
+
+  for (const [letter, freq] of Object.entries(letterPoolDict)) {
+    for (let i = 0; i < freq; i++) {
+      letterPoolList.push(letter);
+    }
+  }
+  return letterPoolList;
+};
+
+const getRandomNumber = (max) => {
+  //Function has one parameter, an integer named max, and returns random number between 0 and max
+  return Math.floor(Math.random() * (max + 1));
+};
+
+const removeLetter = (list, letter) => {
+//Function has two parameters, a list and a letter, and returns a list with the letter removed
+  const index = list.indexOf(letter);
+  list.splice(index, 1);
```

Comment:
These are letter counts will the same size but what if they weren't? What if the input size varied? What would be the time complexity of your code? What CS fundamentals data structure did we learn about that is handy for keeping track of the number of occurrences? Is there anything we can cross reference this data structure to?